### DISCUSSION TITLE
2024-11-15 - Do you know the best email template software? - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by giscus** [SSWConsulting/SSW.Rules.Content#9579] [Chewing the Fat]
Learn about the best email template software options to ensure consistency in your organization's communications.

https://www.ssw.com.au/rules/use-the-best-email-templates/

**Comment by stephanfakossw** [comment]
Great rule and awesome video!

I can see the value in this, and I understand it's a very quick way of addressing heavy admin we all experience...but I think using emails is very retro (E.g. it'll be great to use EasyLeave for leave requests).

The issue with Content Chooser is reliability. 
The changes made in the doc do not automatically find their way into Outlook | Content chooser. Any updates to doc templates need to be manually synced to the content chooser - this can cause templates to be out of date.

**Reply by bettybondoc** [reply]
+1 Prefer to automate things where we can :)

**Comment by sethdaily** [comment]
I have been loving the content chooser email templates. Great experience

**Comment by MarcusIrmscherSSW** [comment]
Great video, Tanya explains email templates so clearly. Really helpful tips!

**Comment by drwharris** [comment]
Great Video Tanya!

Love the fact that we dont have to move between SugarLearning and Outlook.

Would love to see a bit more categorisation on the templates though.

**Comment by TobyChurches** [comment]
Great Video! I've enjoyed using these email templates!
However, specifically for the Daily scrum emails, I still prefer copying from the previous email since a lot of general data stays the same each time I send it

**Reply by babakamyljanovssw** [reply]
+1

**Comment by jakebayliss** [comment]
Seems like a no brainer!

**Comment by Levijj22** [comment]
Looks good! Great video Tanya.

**Comment by awaltos** [comment]
Nice to have templates so easy to access within Outlook. Content Chooser seems to work quite well from a user perspective. 
Would be nice to see categorisation and sentence case for template names instead of kebab, although I loved seeing Favourites - this will help reduce noise for repetitive email tasks. 
Selecting a template adds the content (no surprise there) selecting **another** template adds all the fields to the working draft which is annoying if user just wants to preview the content - user has to delete everything manually when changing template. Would be nice to have working draft cleared and replaced with the new content when selecting a subsequent template.

**Comment by RobJT-wq** [comment]
Neat rule surprised I haven't been using Content Chooser earlier.

**Comment by vladislav-kir** [comment]
Great rule and a video! I only used Content Chooser templates twice so far, but they look quite helpful :)

**Comment by danielmackay** [comment]
Content Chooser is a MASSIVE improvement over having to search through through an ocean of rules, or other internal systems to find the right template. It makes it super easy to answer the question: "Do we have a template for that"?

**Comment by jeoffreyfischer** [comment]
Great video! Email templates made my life so much easier :D

**Comment by StefStar** [comment]
Great video Tanya, haven't been using content chooser much and now I know what I've been missing out on!

**Comment by Calinator444** [comment]
Content chooser can be a huge time saver if you need to send out a quick email. It's nearly to save you having to get a checked by, or when you do get a checked by there's usually minor changes.

**Comment by bahjatalaadelssw** [comment]
Great explainer! Dynamics is a doozy 🤪 I just wish My Templates was bigger that 32kb!

**Comment by Ben0189** [comment]
Great tips and awesome video

**Comment by isaaclombardssw** [comment]
Big time saver, saving these kinds of templates on hand :)

**Comment by Lewkans** [comment]
Great rule and video. I like that I don't have to look through SugarLearning to find the template to copy.

**Comment by sam-s-maher** [comment]
Greater consistency with less clicks - big fan of the content chooser!

**Comment by ncn-ssw** [comment]
Content Chooser does look like an easy tool to use.

**Comment by Marxoz** [comment]
Awesome! So much better than trying to find the right template to send. Also, when you use the Content Chooser you know you have the latest version of the template.

**Comment by steven0x51** [comment]
Great rules! very clear and explanatory

**Comment by chrisschultzssw** [comment]
Content Chooser is a great solution for users.
It seems like a lot of work to set up and maintain the templates though.

**Comment by ricksu978** [comment]
I like the comparison table!
Content Chooser is quick and easy to use.

**Comment by jimmy1992abc** [comment]
Content Chooser is great, but it requires a manual update (few selections and clicks) when the content is updated.
Meanwhile, the content must be presented in SharePoint (not in SugarLearning).

This creates problems with the templates from SugarLearning because the changes from SugarLearning won’t sync to SharePoint easily.
It is also a concern that missing the update step would result in the Content Chooser template being out of date.

I still prefer to use Content Chooser for certain tasks where the templates are unlikely to be changed. However, I would expect a major feature update that can sync the template automatically.

**Comment by MattParkerDev** [comment]
Looks great to not have to open sugarlearning and find the right template!

**Comment by SamBWagner** [comment]
Pretty sweet. Nice to see a solid lil' guide to upgrade your day-to-day workflow. :)

**Comment by PennyWalker** [comment]
This is going to save people a lot of time! I loved how all the options were explained in the video - I learned a lot. I'm so glad I wont have to grab a template from another source and copy and paste anymore. I agree with Stephan about making sure people know how to save an edited template - that could cause problems if not overridden.

**Reply by leahy268** [reply]
It looks as though they have a feature coming that may automatically update it for us.  However if that fails I think a simple flow that reminded people to update the template would solve it.

**Comment by leahy268** [comment]
Great Video.  Could be slightly biased.  But really love having an easy way to find the template I need so quickly.

**Comment by Tanya-Leahy** [comment]
Great comparison of the template options available. Content Chooser is a clear winner and has made life so much easier to look up templates.

**Comment by amankumarrr** [comment]
Great video! Really easy to follow!

I haven't used Content Chooser before, but it looks like a promising tool that lets you work without constantly switching between different platforms, providing a unified space to extract all templates.

**Comment by miichaelsmedley** [comment]
I use a lot of Templates and this plugin is way better than the alternatives. I like that this has been explained here.

**Comment by tkapa** [comment]
I haven't previously been using Email templates, but I might have to start :)

**Comment by bettybondoc** [comment]
Great video and rule! I just used it this morning to send my "Working From Home" email. Worked like a charm. :)

**Comment by babakamyljanovssw** [comment]
Great video and rule!
Just checked out "Content Chooser" and selecting any templates directly in Outlook saves a lot of time 🚀

**Comment by KailinYuan** [comment]
Love how clearly Tanya explains it. Great video！

**Comment by jimmidier** [comment]
Great video and rule! This can save tons of time on drafting emails

**Comment by thom-wang** [comment]
Really loved content chooser in office, fluent experience between Word and Outlook

**Comment by tino-liu** [comment]
Awesome video with great explanation! I checked out Content Chooser and already knew it will save me a lot of time.

**Comment by ClaraFang-SSW** [comment]
It’s incredibly useful and provides clear instructions! In the past, I found it inefficient to go back to Sugarlearning to search for the correct email templates. However, now that we can store templates in multiple ways, it’s much more effective and time-saving.

**Comment by Aibono1225** [comment]
Great video and rule! This seems like an incredibly useful tool that will definitely save a lot of time.

**Comment by suzanne-gibson** [comment]
I really like using Content Chooser templates, not only have they been critical to my progress of SugarLearning but they have been a great learning tool for formatting and writing other emails.
